spec/board_spec.rb

```diff
@@ -81,9 +85,68 @@
 
   describe 'Available squares: ' do
     it 'returns the available squares as an array' do
-      board = Board.new(['x', 2, 'o', 4, 5, 6, 7, 8, 9])
+      board = board_two_moves
+
+      available_squares = board.available_squares
+
+      expect(available_squares).to eq([2, 4, 5, 6, 7, 8, 9])
+    end
+  end
+
+  describe 'Copy board: ' do
+    it 'creates a new instance of Board with the current squares' do
+      board = board_two_moves
+
+      new_board = board.copy_board
 
-      expect(board.available_squares).to eq([2, 4, 5, 6, 7, 8, 9])
+      expect(new_board).to be_an_instance_of(Board)
+      expect(board).not_to eq(new_board)
+      expect(new_board.squares).to eq(['x', 2, 'o', 4, 5, 6, 7, 8, 9])
     end
   end
+
+  describe 'Reset a square: ' do
+    it 'makes the selected square empty' do
+      board = board_two_moves
+      expect(board.squares[0]).to eq('x')
+
+      board.reset_square(1)
+      expect(board.squares[0]).to eq(1)
+    end
+  end
+  describe 'Stop playing: ' do
```

Comment:
Rather than testing the implementation of a method, such as stop playing, could you find a way to test the behaviour instead? It might just be that the test descriptions need to be reworded...

Reply:
Thanks Ashley, I think it's the test description. I'll change this